collect_app/src/main/java/org/odk/collect/android/utilities/FormUtils.java

```diff
@@ -16,6 +18,7 @@ private FormUtils() {
 
     }
 
+    @NonNull
     public static List<File> getMediaFiles(Form form) {
```

Comment:
We should probably add a `@NonNull` to the param as well.

Reply:
Done.